Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -71,6 +80,18 @@ - (NSURLSession *)waitingSession
     return _waitingSession;
 }
 
+- (NSURLSession *)backgroundSession
+{
+    if (!_backgroundSession) {
+        NSURLSessionConfiguration *backgroundSessionConfiguration = [NSURLSessionConfiguration
```

Comment:
again, just for the sake of being consistent in style, would it be possible to add a `createBackgroundSession` method like the `createWaitingSession` one?